.github/workflows/build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - name: 'Install jOOQ Snapshot'
         working-directory: jOOQ
-        run: mvn --no-transfer-progress -DskipTests install
+        run: mvn --no-transfer-progress -DskipTests -pl jOOQ -pl jOOQ-meta -am install
```

Comment:
I don't think you'll need `jOOQ-meta`? That's just the meta data source of the code generator. The `org.jooq.MetaProvider` and `org.jooq.Meta` types and implementations are in `jOOQ`, so that should suffice for this build.